docs/JsonConfigFile-NNPA.md

```diff
@@ -0,0 +1,237 @@
+<!--- SPDX-License-Identifier: Apache-2.0 -->
+
+The compiler provides two options to allow users control how ONNX operators run on NNPA.
+- `--nnpa-load-config-file` to load configuration settings for NNPA from a JSON file.
+- `--nnpa-save-config-file` to save configuration settings for NNPA to a JSON file.
+
+It is common to use ONNX node names to match ONNX operators. If you're unsure about the node names in your model, you can use the `--nnpa-save-config-file` option to generate a configuration file as a starting point.
+
+Alternatively, you can open the ONNX model using a visualizer like Netron to inspect the node names. Note that the actual node names used by the compiler may differ slightly from those shown in Netron due to compiler optimizations, though they are usually the same.
```

Comment:
@AlexandreEichenberger I added some sentences to mention Netron app when looking for node names.